packages/gatsby-cli/src/reporter/index.ts

```diff
@@ -416,15 +480,24 @@ const reporter: Reporter = {
       },
       span,
     }
-  },
+  }
   // This method was called in older versions of gatsby, so we need to keep it to avoid
   // "reporter._setStage is not a function" error when gatsby@<2.16 is used with gatsby-cli@>=2.8
-  _setStage() {},
+  _setStage(): void {}
 }
 
-console.log = (...args) => reporter.log(util.format(...args))
-console.warn = (...args) => reporter.warn(util.format(...args))
-console.info = (...args) => reporter.info(util.format(...args))
-console.error = (...args) => reporter.error(util.format(...args))
+const reporter = new Reporter()
+
+console.log = (format: string, ...args: unknown[]): ILogAction =>
+  reporter.log(util.format(format, ...args))
+console.warn = (format: string, ...args: unknown[]): ILogAction =>
+  reporter.warn(util.format(format, ...args))
+console.info = (format: string, ...args: unknown[]): ILogAction =>
+  reporter.info(util.format(format, ...args))
+console.error = (
+  format: string,
+  ...args: unknown[]
+): IStructuredError | IStructuredError[] =>
+  reporter.error(util.format(format, ...args))
 
 module.exports = reporter
```

Comment:
we should be changing this to a named export

Reply:
I left the export the same because it's a breaking change, and is used in so many places